packages/common/src/utils/certificateUtils.ts

```diff
@@ -111,3 +111,7 @@ export const getComplementedByCertificateRelation = (certificateMetadata: Certif
     (r) => r.type === CertificateRelationType.COMPLEMENTED && r.status !== CertificateStatus.REVOKED
   )
 }
+
+export interface FunctionDisabled {
```

Comment:
I think we should move this to another file. Either something already existing or create a new file for this time of functionality.